### PR TITLE
Re-do the "Media_Links::" processing.

### PR DESCRIPTION
**Pull request by hdurer**
Re-do it as a loop rather than raising an error when the search fails and trying to catch that wholesale.

### CODE REVIEW

File: lisp/mastodon-media.el [hunk 1]
```diff
@@ -42,23 +42,24 @@
...
 
-contains `Media_Links::'."
-  (when start (goto-char (point-min)))
-  (search-forward-regexp "Media_Link::" nil nil nil)
-  (let ((start (progn (move-beginning-of-line '()) (point)))
-	(end (progn (move-end-of-line '()) (point))))
-    (list start end)))
+Returns the cons of (`start' . `end') points of that line or nil no
+more media links were found."
+  (let ((found? (search-forward-regexp "Media_Link::" nil t)))
```
Comment (jdenen) [inline lisp/mastodon-media.el]: Change `found?` to `foundp` to match convention.

https://github.com/bbatsov/emacs-lisp-style-guide#naming

**Review by jdenen — CHANGES_REQUESTED** [review]

File: lisp/mastodon-media.el (hunk 1 above)
Comment (jdenen) [reply lisp/mastodon-media.el]: This is a small nit but I like to follow convention. Otherwise, the PR looks great! Thanks.

**Comment by hdurer** [comment]
Renamed and rebased to latest `develop`.

(Although technically, the style guide only prescribed the name of predicate methods not of boolean values. ☺ )

**Comment by jdenen** [comment]
That's fair. The question mark looked weird in elisp though. 😛

**Review by jdenen — APPROVED** [review]

**Pull request merged**